CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
-# Changelog
+v2.3.0 / 2020-11-14
+========================
+* fix(restore): set targetip on CVRs after restore is completed ([#1761](https://github.com/openebs/maya/pull/1761),[@zlymeda](https://github.com/zlymeda))
+* refactor(localpv): move builds to dynamic-localpv-provisioner repo ([#1762](https://github.com/openebs/maya/pull/1762),[@kmova](https://github.com/kmova))
+* fix(upgrade): remove quay.io as default url-prefix ([#1768](https://github.com/openebs/maya/pull/1768),[@shubham14bajpai](https://github.com/shubham14bajpai))
+* chore(build): add support for multi-arch builds ([#1764](https://github.com/openebs/maya/pull/1764),[@shubham14bajpai](https://github.com/shubham14bajpai))
+* refactor(exporter): move builds to openebs-exporter repo ([#1763](https://github.com/openebs/maya/pull/1763),[@shubham14bajpai](https://github.com/shubham14bajpai))
+* fix(upgrade): allow upgrades for volumes without monitor ([#1760](https://github.com/openebs/maya/pull/1760),[@shubham14bajpai](https://github.com/shubham14bajpai))
+* fix(build): propagate RELEASE_TAG to buildx container ([#1766](https://github.com/openebs/maya/pull/1766),[@shubham14bajpai](https://github.com/shubham14bajpai))
```

Comment:
IMO its not required as a changelog, its covered as part of  `Support for multi arch` change